components/org.wso2.carbon.identity.keyrotation/src/main/java/org.wso2.carbon.identity.keyrotation/model/CipherMetaData.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.keyrotation.model;
+
+import org.apache.axiom.om.util.Base64;
+
+import org.wso2.carbon.identity.keyrotation.util.KeyRotationConstants;
+
+/**
+ * This class holds the cipher, transformation and initialization vector metadata.
+ */
+public class CipherMetaData {
+
+    private String c;
```

Comment:
Can we have descriptive variable for c, t and iv

Reply:
These values are embedded in the self contained object inside the encrypted text, so we cannot change these variable  names. These particular variable names are used in the crypto-service for the encryption and decryption tasks.

Reply:
+1